addons/http_routing/models/ir_http.py

```diff
@@ -91,7 +91,7 @@ def slugify(s, max_length=0, path=False):
 def slug(value):
     try:
         if not value.id:
-            raise ValueError("Cannot slug non-existent record %s" % value)
+            raise exceptions.MissingError(_("This page does not exist"))
```

Comment:
@xmo-odoo @Julien00859 Is this fine as stable change?
Isn't it like changing the return type of a method, indirectly?

Asking as this is quite a mainstream method used probably in custos, is that ok in case someone is "listening" to the `raise` case?
It won't be caught anymore.

Reply:
the boss told us to "go fast" and "break things"

Reply:
You missed the whole point: "go fast, **pay as much attention as you can to not break things but don't get parano about it, and if despite being careful and thinking about what could break you missed something, it's ok to** break things **unintentionally**"

It does not apply here as the possible issue is detected.
But I guess the answer is quite obvious: it's not stable compliant, is it?

Reply:
Message is misleading imo now after your change.
In your own case, that a page... But you could raise this error from other place, with an issue unrelated to a page.

It is like if your replace. `"you cannot have access to %s" % model._name` by an hard coded message `"you cannot have access to this product"` because you have a bug on your ecommerce.